src/box/lua/config/init.lua

```diff
@@ -458,8 +463,14 @@ end
 
 function methods.info(self)
     selfcheck(self, 'info')
+    -- Don't return alert codes in config:info().
+    local alerts = {}
+    for _, alert in pairs(self._alerts) do
+        table.insert(alerts, alert)
+    end
```

Comment:
I would show it sorted in the order of appearance. At least we can use the timestamps to sort the alerts.

Reply:
Fixed